multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -137,6 +137,12 @@
 IA3_BIAS = "ia3_bias"
 IA3_NORM = "ia3_norm"
 
+# DeepSpeed constants (for now gather/reduce size are constants for single GPU)
+DEEPSPEED_OFFLOADING = "deepspeed_stage_3_offload"
+DEEPSPEED_MIN_PL_VERSION = "1.7.7"
+DEEPSPEED_ALLGATHER_SIZE = 1e9
+DEEPSPEED_ALLREDUCE_SIZE = 1e9
```

Comment:
We can put the `ALLGATHER_SIZE` and `ALLREDUCE_SIZE` as configurable options in https://github.com/awslabs/autogluon/blob/master/multimodal/src/autogluon/multimodal/configs/environment/default.yaml .